plsync/mlabconfig_test.py

```diff
@@ -234,6 +295,19 @@ def test_export_mlab_zone_header(self):
 
         self.assertEqual(output.getvalue(), 'before; middle; after')
 
+    @mock.patch('__builtin__.open')
+    def test_export_mlab_server_network_config(self, mock_open):
+        stdout = StringIO.StringIO()
+        name_tmpl = '{{hostname}}-foo.ipxe'
+        input_tmpl = StringIO.StringIO('ip={{ip}} ; echo ${ip}')
+        file_output = StringIO.StringIO()
+        mock_open.return_value = OpenStringIO(file_output)
+
+        mlabconfig.export_mlab_server_network_config(
+            stdout, self.sites, name_tmpl, input_tmpl, 'mlab1.abc01')
+
+        self.assertEqual(file_output.getvalue(), 'ip=192.168.1.9 ; echo ${ip}')
```

Comment:
This isn't necessarily a problem, but just me trying to understand. Since `file_output` isn't passed to `mlabconfig.export_mlab_server_network_config()`, how can `file_output` contain what should ostensibly be written to the file evenutally created by `name_tmpl`?